.github/pull_request_template.md

```diff
@@ -16,5 +16,7 @@
 * [ ] Properly cite in `BibTeX` format
 * [ ] Add passing test(s)
 * [ ] Add test data
+* [ ] Use `lazy_imports` to access external dependencies (if applicable)
+* [ ] Update `setup.py`, `LazyImporter`, and `LazyImportsTest` with dependencies (if applicable)
```

Comment:
Rather than naming some symbols which may be obscure for new contributors, I would instead only add a single item with a link to the doc.
```
* [ ] If using additional dependencies (e.g. `nltk`), use [lazy_imports](https://www.tensorflow.org/datasets/add_dataset#extra_dependencies)
```